working-groups/meta/NOTES.md

```diff
@@ -0,0 +1,36 @@
+# Meta Meeting Notes
+This document contains meeting notes from the Meta working group.
+
+## 2019-02-21:
```

Comment:
Could you link to the Zulip thread for this meeting?

Reply:
Done.